docs/cluster-management/developing-locally.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: main
+title: Configuring the UI
+category: Cluster Management
+menu: menu
+toc:
+    - title: Developing locally
+      url: "#developing-locally"
+      active: true
+    - title: Developing locally with executor-queue
+      url: "#developing-locally-with-executor-queue"
+
+---
+## Developing Locally
```

Comment:
I think it would be good to have a sentence or two highlighting the intent of this doc vs https://docs.screwdriver.cd/cluster-management/running-locally

And it might be good to add a link or something to your new doc from this one: https://docs.screwdriver.cd/about/contributing/getting-started-developing